fury/ui.py

```diff
@@ -1420,6 +1429,30 @@ def _set_position(self, position):
         if self._background is not None:
             self._background.SetPosition(*self.actor.GetPosition())
 
+    @property
+    def position2(self):
+        """
+        Gets top-right corner position.
+
+        Returns
+        -------
+        (float, float, float)
+        """
+        return self.actor.GetPosition2()
+
+    @position2.setter
+    def _set_position2(self, position2=(100, 100, 0)):
+        """ Set text actor position2.
+
+        Parameters
+        ----------
+        position : (float, float)
+            The new position. (x, y) in pixels.
+        """
+        self.actor.SetPosition2(*position2)
+        if self._background is not None:
+            self._background.SetPosition2(*self.actor.GetPosition2())
+
     def _get_size(self):
```

Comment:
You should update `_get_size`

Reply:
Sure, also updating this will most likely break the ListBoxItem2D text-overflow fix. So should I fix that within this PR itself?